dm/dm/worker/source_worker.go

```diff
@@ -1338,15 +1338,18 @@ func (w *SourceWorker) CheckCfgCanUpdated(cfg *config.SubTaskConfig) error {
 	return subTask.CheckUnitCfgCanUpdate(cfg)
 }
 
-func (w *SourceWorker) GetWorkerValidatorErr(taskName string, errState pb.ValidateErrorState) []*pb.ValidationError {
+func (w *SourceWorker) GetWorkerValidatorErr(taskName string, errState pb.ValidateErrorState) ([]*pb.ValidationError, error) {
```

Comment:
we don't need to lock worker here, OperateWorkerValidatorErr too

Reply:
address in https://github.com/pingcap/tiflow/commit/5726353f3b7d20b3d383a52d0fb91fc2313714dd